src/components/sectionTypes/sectionTypes.tsx

```diff
@@ -5,36 +5,58 @@ import cat2 from '../../images/cat2.png';
 import cat3 from '../../images/cat3.png';
 import cat4 from '../../images/cat4.png';
 import cat5 from '../../images/cat5.png';
+import cat6 from '../../images/cat6.png';
 
 
-interface SectionTypesProps { }
+interface SectionTypesProps {
+  onCategory: (e: any) => void
+}
+
+
+const SectionTypes: FC<SectionTypesProps> = (props: SectionTypesProps) => {
+
 
-const SectionTypes: FC<SectionTypesProps> = () => {
   return (
     <div className={styles.sectionTypes} data-testid="sectionTypes">
       <div className={styles.title}>
         <h2>Category products</h2>
       </div>
       <div className={styles.categories}>
         <figure>
-          <img src={cat1} alt="animals" className={styles.category} />
-          <figcaption>Animals</figcaption>
+          <button onClick={props.onCategory} >
```

Comment:
yo creo que esto se puede reducir en un budle for, si metes todo lo que cambia en objeto e iteras por el
que opinas?
al final lo que cambia es lo de cat1, cat2, etc... y el string que marca el tipo y el alt (que al final uno es el string capitalizado del otro no?)

Reply:
lo miro